doc/source/text.rst

```diff
@@ -199,6 +199,143 @@ regular expression object will raise a ``ValueError``.
     ---------------------------------------------------------------------------
     ValueError: case and flags cannot be set when pat is a compiled regex
 
+.. _text.concatenate:
+
+Concatenation
+-------------
+
+There are several ways to concatenate a ``Series``, either with itself or others, all based on :meth:`~Series.str.cat`.
+
+Concatenating a single Series into a string
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The content of a ``Series`` can be concatenated:
+
+.. ipython:: python
+
+    s = pd.Series(['a', 'b', 'c', 'd'])
+    s.str.cat(sep=',')
+    
+If not specified, the keyword ``sep`` for the separator defaults to ``''``:
+
+.. ipython:: python
+
+    s.str.cat()
+
+By default, missing data is ignored. Using ``na_rep``, they can be given a representation:
+
+.. ipython:: python
+
+    t = pd.Series(['a', 'b', np.nan, 'd'])
+    t.str.cat(sep=',')
+    t.str.cat(sep=',', na_rep='-')
+
+Concatenating a Series and something one-dimensional into a Series
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first argument to :meth:`~Series.str.cat` (which corresponds to the keyword ``others``) can be used to specify values that should be concatenated correspondingly. 
+The result will be a ``Series`` of strings. Possible input types are ``list``, ``Series`` and ``np.ndarray``, all of which must match the length of
+the calling ``Series``.
+
+.. ipython:: python
+
+    s.str.cat(['A', 'B', 'C', 'D'])
+    s.str.cat(s, sep = ':')
+    s.str.cat(s.values, sep = '.')
+    
+Missing values will remain missing in the result, *unless* ``na_rep`` is specified:
```

Comment:
you already showed this above, it is redundant here

Reply:
It's different, because for ``t.str.cat()``, NaNs dissapear completely, while in the case of two Series, the value becomes NaN (overriding potentially relevant nonnull information in the other Series).